sdb/commands/zfs/range_tree.py

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+from typing import Iterable
+
+import drgn
+import sdb
+from sdb.commands.zfs.btree import Btree
+from sdb.commands.cast import Cast
+
+
+class RangeTree(sdb.PrettyPrinter):
+    names = ['range_tree']
+    input_type = 'range_tree_t *'
+
+    def pretty_print(self, objs: Iterable[drgn.Object]) -> None:
+
+        class RangeTreeSeg(sdb.Command):
```

Comment:
It would be nice to write a comment about the reason that this is embedded here.
On that context, I liked your description in the PR about this:
```
One complication is that range_seg*_t's specify their ranges
relative to rt_start/rt_shift, which are not accessible from the
range_seg*_t. Therefore, there is no accessible pretty-printer for
range seg's. They can only be pretty-printed as part of a
range_tree_t*, from the range_tree pretty-printer.
```

It may also be nice to just note the things that we talked about
offline about this interacting with `all_commands`.